CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Fixed typos in header-related SCSS variable names.
 - Fix non-conformities in blogpost, category, organization & person glimpses.
-- Fix lodash-es, handlebars vulnerabilities by upgrading versions.
+- Fix lodash, lodash-es, handlebars vulnerabilities by upgrading versions.
```

Comment:
Should have been in a "security" section, see https://keepachangelog.com/en/1.0.0/